examples/zero_copy/Chatter.msg

```diff
@@ -0,0 +1,3 @@
+uint64 timestamp
+char[4000000] data # shared memory backend can only deal with fixed size messages
```

Comment:
Would it make sense to put here 4MiB, aka 4 * 1024 * 1024?

Reply:
4MiB exceeded their default reserved shared memory pool size, I used 3 MiB instead.